components/org.wso2.carbon.identity.sts.mgt/src/main/java/org/wso2/carbon/identity/sts/mgt/internal/IdentitySTSMgtServiceComponent.java

```diff
@@ -279,10 +282,14 @@ private void loadSecurityScenarios() throws Exception {
 
                 // Store the scenario in the Registry
                 if (!scenarioId.equals(SecurityConstants.SCENARIO_DISABLE_SECURITY) &&
-                    !scenarioId.equals(SecurityConstants.POLICY_FROM_REG_SCENARIO)) {
+                        !scenarioId.equals(SecurityConstants.POLICY_FROM_REG_SCENARIO)) {
                     Resource scenarioResource = new ResourceImpl();
+                    URL urlResource = bundleContext.getBundle().getResource(scenarioId + "-policy.xml");
+                    if (urlResource == null) {
+                        return;
+                    }
                     scenarioResource.setContentStream(
-                            bundleContext.getBundle().getResource(scenarioId + "-policy.xml").openStream());
+                            urlResource.openStream());
```

Comment:
can we get this to one line.

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/1303

Reply:
It has been fixed with https://github.com/wso2-extensions/identity-inbound-auth-sts/pull/63/files#diff-dfe5fbb75fd0b55201fb57346f45b229R289